distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Split_Tokenize.enso

```diff
@@ -0,0 +1,246 @@
+
+            builders.to_vector
+
+        _ : Integer ->
+            builders = Vector.new column_count (_-> make_string_builder num_rows)
+
+            output_lengths = 0.up_to num_rows . map i->
+                input_value = input_storage.getItemBoxed i
+                output_values = function input_value
+
+                ## Add `output_values` to builders; if there are more builders
+                   than `output_values`, pad with null.
+                0.up_to builders.length . map i->
+                    builders.at i . appendNoGrow (output_values.get i Nothing)
+
+                output_values.length
+
+            max_output_length = output_lengths.fold 0 .max
+
+            if max_output_length > column_count then
+                problem = Column_Count_Exceeded.Error column_count max_output_length
+                problem_builder.report_other_warning problem
+
+            builders
+
+    # Build Columns.
+    builders.map .seal . map_with_index i->storage->
+        name = input_column.name + "_" + i.to_text
+        Column.from_storage name storage
+
+## PRIVATE
+   Name a set of column vectors to be unique when added to a table.
+rename_new_columns : Table -> Vector Column -> Problem_Builder -> Vector Column
+rename_new_columns table columns problem_builder =
+    unique = Unique_Name_Strategy.new
+    unique.mark_used <| table.columns.map .name
+    new_columns = columns.map column->
+        new_name = unique.make_unique column.name
+        column.rename new_name
+    problem_builder.report_unique_name_strategy unique
+    new_columns
+
+## PRIVATE
+   Replace a single column in a table with new columns.
+replace_column_with_columns : Table -> Column -> Vector Column -> Table
+replace_column_with_columns table old_column new_columns =
+    Table.new ((table.columns.map (c-> if c.name == old_column.name then new_columns else [c])).flatten)
+
+## PRIVATE
+   Return the maximum value of the vector.
+   Throws Empty_Error if the vector is empty.
+maximum : Vector Any -> Vector Any
+maximum vec = if vec.is_empty then Nothing else
+    vec.reduce (a-> b-> a.max b)
+
+## PRIVATE
+   Wrap a function so that it returns Nothing if passed Nothing
+ignore_nothing : (Any -> Any) -> (Any -> Any)
+ignore_nothing function = x-> case x of
+    _ : Nothing -> Nothing
+    _ -> function x
+
+## PRIVATE
+   Asserts that a column exists in the table and is a text column.
+expect_text_column : Table -> Text | Integer -> Problem_Behavior -> Any -> Any
+expect_text_column table column_id ~action =
+    column = table.get column_id
+    case column of
+        Nothing ->
+            Error.throw <| Missing_Input_Columns.Error [column_id]
+        _ : Column ->
+            case Value_Type.is_text column.value_type of
+                False ->
+                    Error.throw <| Invalid_Value_Type.Error Value_Type.Char column.value_type column_id
+                True ->
+                    action
```

Comment:
This method is not needed IMO. It essentially reimplements what `Table.at`+`Value_Type.expect_text` will do. Please use these instead to avoid reinventing the wheel, unless there is some deeper reason to do it.

Reply:
Removed.